server/notification-providers/ntfy.js

```diff
@@ -52,15 +52,15 @@ class Ntfy extends NotificationProvider {
                 "topic": notification.ntfytopic,
                 "message": heartbeatJSON.msg,
                 "priority": priority,
-                "title": monitorJSON.name + " " + status + " [Uptime-Kuma]",
+                "title": monitorJSON.pathName + " " + status + " [Uptime-Kuma]",
                 "tags": tags,
             };
 
             if (monitorJSON.url && monitorJSON.url !== "https://") {
                 data.actions = [
                     {
                         "action": "view",
-                        "label": "Open " + monitorJSON.name,
+                        "label": "Open " + monitorJSON.pathName,
```

Comment:
no idea if special characters are allowed here

```suggestion
                        "label": "Open " + monitorJSON.name,
```